Makefile

```diff
@@ -16,9 +16,9 @@ clean:
 build:
 	cd $(GENERATOR_FOLDER) && swift build $(SWIFT_BUILD_FLAGS)
 
-install: uninstall clean build
+install: uninstall clean build archive_generator
 	install -d "$(BINARY_FOLDER)"
-	install "$(GENERATOR_ARCHIVE_PATH)" "$(BINARY_FOLDER)"
```

Comment:
`GENERATOR_ARCHIVE_PATH` is no longer needed (see line 4) after this change. Also means that the `swift build --show-bin-path` is no longer being used, and we have `bin` hardcoded. That part if probably fine, can't remember when we added the show-bin-path thing.

Reply:
`GENERATOR_ARCHIVE_PATH` is still used in `archive_generator` command.

Reply:
Oh, yeah, nevermind.  I used the "expand" icon on both above and below the change and looked, and now I realize you have to click expand several times to get to the end of the file. Will remember that next time.